parking-lot-help-request/README.md

```diff
@@ -0,0 +1,118 @@
+    1. Red wire -> `AUX1` on the Notecarrier A
+    2. Black wire -> `-` (negative bus) on the breadboard
+![Button's red wire to the Notecarrier A's AUX1 pin and black wire to the breadboard's negative bus](images/readme-add-button.jpg)
+
+1. Plug the LiPo battery and solar panel into the Notecarrier A's JST connectors for each piece of hardware
+![Finished hardware](images/readme-finished-hardware.jpg)
+_This hardware is assembled: Notecarrier, Notecard, LED, button, LiPo battery, and solar panel._
+1. Now it's time to configure Notehub and the Notecard!
+
+## Notehub
+
+The Parking Lot Help Request project runs with the help of Notehub. As such, to run this app you’ll need to create a Notehub account, and start a new Notehub project.
+
+### Creating a New Project
+
+1. Sign up for a free [Notehub](https://notehub.io) account, if you don’t have one
+already.
+1. Click the blue **Create Project** button in the top-right corner of the screen.
+1. Give the project a name, such as “ParkingLotHelpRequest”, and then click the next
+**Create Project** button.
+
+![Creating a project in Notehub with a name](images/readme-notehub-project.png)
+
+Once your project's created, copy the product UID down somewhere: you'll need it to program the Notecard in the next section.
+
+### Optional: Grouping Devices By Fleet
+
+Notehub has the option to store groups of devices by fleets, which is useful 
+for organizing similar groups of devices and storing project settings that need to be shared and synchronized across devices.
+
+For this project, you can group devices by something like "Lot Location", simply by updating a project's [fleet](https://dev.blues.io/reference/glossary/#fleet)
+name, and assigning devices to that fleet.
+
+Notehub creates an initial default fleet for you when you start a new project, and you
+can use that as your starting point.
+
+1. Inside your Notehub project, navigate to your fleet in the navigation menu, and then click the **Settings** link.
+![Location of Notehub fleet settings](images/readme-notehub-fleets.png)
+
+1. Inside Settings, change the Fleet name input to something more useful (like "Lot L" for all devices in that particulat lot)
+![How to rename a fleet name in Notehub](images/readme-notehub-fleet-names.png)
+
+1. Then click on the **Devices** link to see all the devices associated with that Notehub project, and simply update the fleet that device is assigned to by using the dropdown.
+![Assign device to particular fleet in Notehub](images/readme-notehub-device-to-fleet.png)
+
+1. You can also rename the device from this page if you need to update its parking spot number in the same way.
+
+With this your Notehub backend is fully configured, and you’re ready to start
+setting up your Notecard.
+
+
+## Notecard Firmware
+
+Although this is a "host-free" project because the Notecard itself can be configured to send notes to Notehub when the button is pressed, the Notecard still needs to be programmed to do so.
```

Comment:
As a suggestion (and this is possibly for the Hackster writeup and not here), it might be worth being very explicit about what host-free means, and getting excited about it—because it’s kind of cool you can do all of this without a host, and you can save $$$ if you’re putting a bunch of these in parking lots.

Reply:
Good call. I'll emphasize more